lms/services/vitalsource/_client.py

```diff
@@ -45,9 +45,15 @@ def get_book_info(self, book_id: str):
 
             raise
 
-        return _BookInfoSchema(response).parse()
+        book_info = _BookInfoSchema(response).parse()
 
-    def get_book_toc(self, book_id: str):
+        return {
+            "id": book_info["vbid"],
+            "title": book_info["title"],
+            "cover_image": book_info["resource_links"]["cover_image"],
+        }
+
+    def get_table_of_contents(self, book_id: str):
```

Comment:
I changed the name here to reflect the name it has all the way down the stack from API view to here.